### DISCUSSION TITLE
Nonexistent function 'set_zoom_margin' in base 'Node2D (PhantomCamera2D)'

### DISCUSSION THREAD

**Discussion by ChildishGiant** [ramokz/phantom-camera#198] [Question / Help Wanted]
Have an array of nodes I want to keep in frame so this seems perfect but for some reason the "set_zoom_margin" call is failing. The exact wording given is `Invalid call. Nonexistent function 'set_zoom_margin' in base 'Node2D (PhantomCamera2D)'.`

```gdscript
	# Zoom camera to fit
	var pcam := $PhantomCamera2D 
	pcam.append_follow_group_node_array(tiles)
	pcam.set_zoom_margin(Vector4(10, 30, 10, 40)) #  <- Errors here
	pcam.set_auto_zoom(true)
```
![image](https://github.com/ramokz/phantom-camera/assets/13716824/6d9b1f5c-043c-4bc1-8e89-3d5545ac42bf)

Thanks in advance 😄

**Comment by ramokz** [comment]
Ah good catch! That's a typo in the documentation.
It's saying `set_zoom_margin()`, but it should be `set_zoom_auto_margin()` — forgot the "auto" bit.

Should be updated on [the site as well now](https://phantom-camera.dev/follow-modes/group#auto-zoom-margin-(2d)-setter).
